src/devices/src/virtio/net/device.rs

```diff
@@ -86,6 +88,9 @@ pub struct Net {
     // Remove once MRG_RXBUF is supported and this variable is actually used.
     #[allow(dead_code)]
     mmds_ns: Option<MmdsNetworkStack>,
+
+    #[cfg(test)]
+    test_mutators: tests::TestMutators,
```

Comment:
Since we create test mutators, why not also use them for checking bad_activate, reads & writes that we would otherwise test using the global metrics object? This way we can move a bit closer to the goal of running the tests in parallel.